core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentUpdateStatusManager.java

```diff
@@ -724,6 +724,7 @@ public void writeLoadDetailsIntoFile(List<SegmentUpdateDetails> listOfSegmentUpd
       brWriter.write(metadataInstance);
     } catch (IOException ioe) {
       LOG.error("Error message: " + ioe.getLocalizedMessage());
+      fileWrite.setFailed();
```

Comment:
At some places IO exception is thrown back after setting the flag and some places exception is not thrown..any specific reason for this?...can we keep the behavior same..better to throw back the exception at all places